apps/framework-cli/tests/python/simple.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import List, Optional
+
+
+type Key[T: (str, int)] = T 
```

Comment:
I fed it to `mypy` and got `simple.py:6: error: invalid syntax  [syntax]`

Reply:
This works

```python
from typing import List, Optional, TypeVar, Union
from typing_extensions import TypeAlias

T = TypeVar('T', bound= Union[str, int])
Key: TypeAlias = T
```

Reply:
Yeah that's the old syntax. The above works in 3.12

Reply:
We can add additional test cases and supported parsings. I'll keep 3.12 for now

Reply:
The python code runs, but mypy 1.10 on python 3.12 rejects it. Is there an alternative type checker?